lib/OpenLayers/Format/WFST/v1_0_0.js

```diff
@@ -139,7 +139,7 @@ OpenLayers.Format.WFST.v1_0_0 = OpenLayers.Class(
                 var prefix = options.featurePrefix;
                 var node = this.createElementNSPlus("wfs:Query", {
                     attributes: {
-                        typeName: (prefix ? prefix + ":" : "") +
+                        typeName: (options.featureNS ? prefix + ":" : "") +
```

Comment:
These look like changes that are unrelated to your Python 3 work.  It is better if you create a dedicated branch for your pull request instead of using `master`.